docs/ides/vscode-extensions.md

```diff
@@ -0,0 +1,131 @@
+# VS Code extensions
+
+This article will show you the ways to add VS Code extensions and use them with
+a Coder workspace:
+
+1. Using the
+   [public extensions marketplaces](vs-code-extensions.md#using-the-public-extensions-marketplaces)
+   with Code Web (code-server)
+1. Adding
+   [extensions to custom images](vs-code-extensions.md#adding-extensions-to-custom-images)
+1. Installing extensions
+   [using its `vsix` file at the command line](vs-code-extensions.md#installing-extensions-using-its-vsix-file-at-the-command-line)
+1. Installing extensions
+   [from a marketplace using the command line](vs-code-extensions.md#installing-from-a-marketplace-at-the-command-line)
+1. Using a
+   [local VS Code instance with SSH](vs-code-extensions.md#using-a-local-vs-code-instance-with-ssh)
+
+## Using the public extensions marketplaces
+
+You can manually add an extension while you're working in the Code Web IDE. The
+extensions can be from Coder's public marketplace, Eclipse Open VSX's public
+marketplace, or the Eclipse Open VSX _local_ marketplace.
+
+![Code Web Extensions](../images/ides/code-web-extensions.png)
+
+> Note: code-server cannot legally connect to Microsoft's public
```

Comment:
Maybe reword is that Microsoft doesn't allow connecting any other vscode based idea to use their marketplace.

Reply:
updated